app/utils/initialize_graceful_exit.js

```diff
@@ -0,0 +1,12 @@
+const config = require('../config');
+const logger = require('../services/logger');
+
+module.exports = () => {
+  process.once('SIGUSR2', () => {
+    process.kill(process.pid, 'SIGUSR2');
```

Comment:
why you kill process here?

Reply:
I think in this place later we can use this [library](https://github.com/emostar/express-graceful-exit) or write something else to grace stop our server or close all socket connections in the app.  Maybe it doesn't the best place listen to this event. I'm thinking of moving the code to restart logger streams to the logger file, but where I should listen `SIGUSR2` to kill the process?